python/src/trezorlib/cli/solana.py

```diff
@@ -0,0 +1,56 @@
+from typing import TYPE_CHECKING
+
+import click
+
+from .. import messages, solana, tools
+from . import with_client
+
+if TYPE_CHECKING:
+    from ..client import TrezorClient
+
+PATH_HELP = "BIP-32 path to key, e.g. m/44'/501'/0'"
+
+
+@click.group(name="solana")
+def cli() -> None:
+    """Solana commands."""
+
+
+@cli.command()
+@click.option("-n", "--address", required=True, help=PATH_HELP)
+@with_client
+def get_public_key(
+    client: "TrezorClient",
+    address: str,
+) -> messages.SolanaPublicKey:
+    """Get Solana public key."""
+    address_n = tools.parse_path(address)
+    return solana.get_public_key(client, address_n)
+
+
+@cli.command()
+@click.option("-n", "--address", required=True, help=PATH_HELP)
+@click.option("-d", "--show-display", is_flag=True)
+@with_client
+def get_address(
+    client: "TrezorClient",
+    address: str,
+    show_display: bool,
+) -> messages.SolanaAddress:
+    """Get Solana address."""
+    address_n = tools.parse_path(address)
+    return solana.get_address(client, address_n, show_display)
+
+
+@cli.command()
+@click.option("-n", "--address", required=True, help=PATH_HELP)
+@click.option("-t", "--serialized-tx", required=True)
```

Comment:
change to `@click.argument`

and honestly maybe add a default for `-n` so that `trezorctl solana sign-tx LAKSJFALKSJF"LASJ` just works? same for `get-address`

Reply:
https://github.com/trezor/trezor-firmware/pull/3359/commits/d09243c566a1fde90faeaa22c1a87eca740d43ef